tests/GoodsTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+use PHPUnit\Framework\TestCase;
+use Stock\Models\Goods;
+
+class GoodsTest extends TestCase
+{
+    public $goods_data = [];
+
+    public function setUp()
+    {
+        $this->goods_data = ['name' => 'brakes', 'price' => 199.00, 'producer' => 'acme', 'quantity' => 20];
+    }
+
+
+    public function testCheckTypeMethod()
+    {
+        $goods = new Goods($this->goods_data);
```

Comment:
W tym konkretnym przypadku nie potrzebujesz aby Twój "Goods" posiadał tożsamość.
Powinieneś stworzyć pusty obiekt "Goods".

`$goods = new Goods():`